js.js

```diff
@@ -0,0 +1,145 @@
+const choices = ['rock', 'paper', 'scissor'];
+
+/**
+ * randomly choose a number from given array
+*/
+function getRandomFromList(arr){
+    const randomIndex = Math.floor(Math.random() * arr.length);
+    const random = arr[randomIndex];
+    return random;
+}
+
+/**
+ * 
+ * @param {*} personChoice - person choice
+ * @param {*} computerChoice - user computer
+ * @param {*} choices - rock | paper | scissor
+ * @returns returns games results
+ * 
+ * implements game logic of rock paper and scissor
+*/
+function RockPaperScissor (personChoice, computerChoice, choices){
```

Comment:
best practice is to avoid nested ifs where possible and use && or || operators for complex conditions.\
The goal should be to make the code readable and easily understandable.